### PR TITLE
build: update to Bazel 0.20

### PR DESCRIPTION
**Pull request by alexeagle**
Introduce a .bazelrc.user file for user- or team-specific settings

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# Load any settings specific to the current user
```
Comment (mgechev) [inline .bazelrc]: https://github.com/angular/angular/pull/27358 introduces this change.

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 /dist/
 /bazel-*
+.bazelrc.user
```
Comment (mgechev) [inline .gitignore]: https://github.com/angular/angular/pull/27358 introduces this.

**Review by mgechev — COMMENTED** [review]
LGTM, left two comments related to the changes that https://github.com/angular/angular/pull/27358 introduces.

**Comment by mgechev** [comment]
Few CIs are failing lint complains for an environment variable, test for a missing dependency, same for build-packages-dist.

**Comment by alexeagle** [comment]
The lint job is blocked on switching from skylint (Java) to buildifier (binary release)
https://github.com/angular/angular/pull/27489

**Review by mgechev — APPROVED** [review]

**Pull request closed without merging**